.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "Launch Extension",
```

Comment:
This debug configuration option allows to easily test the extension in development. It essentially opens another vscode instance with this plugin loaded.

Reply:
This won't cascade to the final user if that's what you mean.